### PR TITLE
delete other twitter

### PR DESCRIPTION
**Pull request by wincowgerDEV**
No need for social media to move to virtual reality communication only.

### CODE REVIEW

**Comment by wincowgerDEV** [comment]
@shelly-mippr can you take a look.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Microplastics Data Portal
 [![MIT License](https://img.shields.io/apm/l/atomic-design-ui.svg?)](https://github.com/tterb/atomic-design-ui/blob/master/LICENSEs)
-[![Twitter Follow](https://img.shields.io/twitter/follow/ThePlastiverse?style=social)](https://twitter.com/ThePlastiverse)
```
Comment (wincowgerDEV) [inline README.md:3]: Where is the metaverse link?

**Review by hannahhapich — APPROVED** [review]

**Review by wincowgerDEV — COMMENTED** [review]
no link

**Review by Richard-Nelson-CalEPA — APPROVED** [review]

**Review by shelly-mippr — APPROVED** [review]
Looks great!

**Pull request merged**